dace/transformation/passes/fusion_inline.py

```diff
@@ -82,3 +83,43 @@ def apply_pass(self, sdfg: SDFG, _: Dict[str, Any]) -> Optional[int]:
 
     def report(self, pass_retval: int) -> str:
         return f'Inlined {pass_retval} SDFGs.'
+
+
+@dataclass(unsafe_hash=True)
+@properties.make_properties
+class FixNestedSDFGReferences(ppl.Pass):
+    """
+    Fixes nested SDFG references to parent state/SDFG/node
+    """
+
+    CATEGORY: str = 'Simplification'
```

Comment:
It may make sense to introduce a category called 'Housekeeping' or something simliar for passes like this. I assume it will not be the last one that doesn't 'simplify' anything, just cleans stuff up.